pkg/task/common.go

```diff
@@ -116,6 +118,9 @@ func DefineCommonFlags(flags *pflag.FlagSet) {
 	flags.Uint64(flagRateLimitUnit, utils.MB, "The unit of rate limit")
 	_ = flags.MarkHidden(flagRateLimitUnit)
 
+	flags.Bool(flagCheckRequirement, true,
```

Comment:
Boolean flags seems must use the form `--some=false` to disable(see [here](https://golang.org/pkg/flag/#hdr-Command_line_flag_syntax)), otherwise when it appears, it would be set to `true`.
Maybe `--skip-check-reqirements` here would be better? Since we must use `--check-requirements=false` to skip check requirements if we let the default value be `true` on this flag.

Reply:
IMO this is fine since we've got `--checksum=false` already. 

`--skip-check-requirements` would be a "negative flag".